apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.jsx

```diff
@@ -451,6 +452,19 @@ const SwapForm = () => {
     // eslint-disable-next-line
   }, [exchangeRate]);
 
+  const debouncedSetFromAmount = useMemo(
+    () =>
+      debounce((amount: BigNumber) => {
+        setNavigation(null);
+        setShowDetails(false);
+        swapTransaction.setFromAmount(amount);
+      }, 400),
+    // cannot depend on swapTransaction as it'll change when new `rates` are fetched
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [],
+    [swapTransaction.setFromAmount],
```

Comment:
Can we remove [swapTransaction.setFromAmount]?

Reply:
https://github.com/LedgerHQ/ledger-live/pull/2238/commits/90982f4f61cf1452864517bcd38cd15236c1789e